profile.md

```diff
@@ -214,13 +228,30 @@ The following properties are defined for usage within a Kubernetes deployment:
   "namespace": "testing"
   ```
 
-The following table specifies which properties MUST appear in each API:
+
+- `clusterID`
+
+  Version: 2.13
```

Comment:
2.13 is out already, 2.14? Or can this profile change at any time?

Reply:
doi! fixed